dydx-testnet-4/gentx/gentx-CryptoCrew.json

```diff
@@ -0,0 +1 @@
+{"body":{"messages":[{"@type":"/cosmos.staking.v1beta1.MsgCreateValidator","description":{"moniker":"✅ CryptoCrew Validators","identity":"9AE70F9E3EDA8956","website":"https://ccvalidators.com","security_contact":"support@ccvalidators.com","details":"We power the interchain. | ccvalidators.com"},"commission":{"rate":"0.050000000000000000","max_rate":"0.200000000000000000","max_change_rate":"0.010000000000000000"},"min_self_delegation":"1","delegator_address":"dydx199fjq4rnfvz24cktl8cervx8h8e90ruk5644xz","validator_address":"dydxvaloper199fjq4rnfvz24cktl8cervx8h8e90ruk3yrrdn","pubkey":{"@type":"/cosmos.crypto.ed25519.PubKey","key":"T1EOKbLQLop4D3+TPiuRJJ22/jAC5UnZ5OrtH6pWLFQ="},"value":{"denom":"adv4tnt","amount":"1000000000000000000000000"}}],"memo":"a1e7b552513a6d75b0d5e157d9e17b1fd575cc29@192.168.0.105:26656","timeout_height":"0","extension_options":[],"non_critical_extension_options":[]},"auth_info":{"signer_infos":[{"public_key":{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"AohdKGNApyLaYEXja/nHmISDWDqied1WAp7jz9HSzQ7n"},"mode_info":{"single":{"mode":"SIGN_MODE_DIRECT"}},"sequence":"0"}],"fee":{"amount":[],"gas_limit":"200000","payer":"","granter":""},"tip":null},"signatures":["XIM8Pp6l1xrtcJsnJex10f8q9lU6ZAYCCxjUy9qcA7dGu0P11XaCZNaRwLGH8TAUV6X/jmjFj8BDCQGlDwxbuQ=="]}
```

Comment:
Hi, @clemensgg, as per the [Prepare for Genesis](https://v4-teacher.vercel.app/validators/preparing_for_genesis), the self-delegation amount is `500000000000000000000000adv4tnt`

Reply:
Thank you @2dpetkov, we have updated the PR as per requirements